library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts

```diff
@@ -1167,17 +1167,20 @@ export class ContentStripWithTabsComponent extends WidgetBaseComponent
     })
     // this.getSelectedIndex(1)
     return [
-    { value: 'all', widgets: this.transformContentsToWidgets(all, strip) },
-    { value: 'upcoming', widgets: this.transformContentsToWidgets(upcoming, strip) },
-    { value: 'overdue', widgets: this.transformContentsToWidgets(overdue, strip) }]
+      { value: 'upcoming', widgets: this.transformContentsToWidgets(upcoming, strip) },
+      { value: 'overdue', widgets: this.transformContentsToWidgets(overdue, strip) },
+      { value: 'completed', widgets: this.transformContentsToWidgets(allCompleted, strip) },
+      { value: 'all', widgets: this.transformContentsToWidgets(all, strip) }]
   }
 
   getSelectedIndex(stripsResultDataMap: any, key: any): number {
     let returnValue = 0
     if (key === 'cbpPlan') {
       if (stripsResultDataMap.tabs.length) {
-        const data = stripsResultDataMap.tabs.filter((ele: any) => ele.value === 'upcoming')
-        returnValue = data[0].widgets && data[0].widgets.length > 0 ? 1 : 0
+        // const data = stripsResultDataMap.tabs.filter((ele: any) => ele.value === 'upcoming')
+        // returnValue = data[0].widgets && data[0].widgets.length > 0 ? 1 : 0
+        const data = stripsResultDataMap.tabs.filter((ele: any) => ele.value === 'upcoming' || ele.value === 'overdue')
+        returnValue = (data[0].widgets && data[0].widgets.length > 0) ? 0 : (data[1].widgets && data[1].widgets.length > 0) ? 1 : 2
```

Comment:
please check if data available then use data[0]

Reply:
done this changes